app-misc/fff/fff-0.0~git.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY=":rocket: fucking fast file-manager"
```

Comment:
I think we should ban explicit profanity from SUMMARY/DESCRIPTION. Haiku's package manager should be a mostly-professional environment language-wise, IMO.

Reply:
I did think of those, but I couldn't find anything on them, so I just copied it from the package's info (`:rocket:` was in the man page for some reason). I've removed them from the recipe now.